src/test/java/team4/footwithme/team/service/TeamServiceImplTest.java

```diff
@@ -0,0 +1,107 @@
+package team4.footwithme.team.service;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.transaction.annotation.Transactional;
+import team4.footwithme.member.domain.*;
+import team4.footwithme.member.repository.MemberRepository;
+import team4.footwithme.team.api.request.TeamAddResquest;
+import team4.footwithme.team.domain.Team;
+import team4.footwithme.team.domain.TeamMember;
+import team4.footwithme.team.domain.TeamMemberRole;
+import team4.footwithme.team.domain.TeamRate;
+import team4.footwithme.team.repository.TeamMemberRepository;
+import team4.footwithme.team.repository.TeamRateRepository;
+import team4.footwithme.team.repository.TeamRepository;
+import team4.footwithme.team.service.response.TeamCreatedResponse;
+import team4.footwithme.team.service.response.TeamInfoResponse;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@SpringBootTest
+class TeamServiceImplTest {
```

Comment:
재경님께도 드렸던 답변인데 
제가 IntegerationTestSupport라는 추상클래스를 만들어두었는데 이거 활용하셔서 @SpringBootTest 어노테이션 지우고 사용하셔도 좋아요
@SpringBootTest를 모든 테스트마다 붙여서 사용하면 테스트할때마다 스프링부트가 실행되어서 느려진다고 하더라구요 환경을 통합해서 부트를 한번만 키는 방식으로 변경해도 좋을 것 같아요

[[Spring] 테스트 환경](https://tommykim.tistory.com/87) 이 글 참고해보시면 좋을 것 같아요

```suggestion
class TeamServiceImplTest extends IntegrationTestSupport {
```

이러한 형태로 사용해보셔도 될 것 같아요

Reply:
오오 맞아요 테스트가 굉장히 느린 느낌이였습니다!! 글 참고해보고 적용시켜 보겠습니다!!